gneiss/sort.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+import pandas as pd
+from functools import partial
+from gneiss.util import match
+
+
+def mean_niche_estimator(abundances, gradient):
+    """ Estimates the mean niche of an organism.
+
+    Calculates the mean niche or an organism along a gradient.
+    This is done by calculating the expected value of an organism
+    across the gradient.
```

Comment:
Since it is a simple mathematical function, can you add it in the documentation? I think in skbio they allow embedding latex-style syntax so it may be worth adding it here too.